Sunrise/ViewModels/BaseViewModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) 2016 Commercetools. All rights reserved.
+//
+
+import ReactiveCocoa
+import Result
+import ObjectMapper
+import Commercetools
+
+class BaseViewModel {
+
+    // Outputs
+    let alertMessageSignal: Signal<String, NoError>
+
+    let alertMessageObserver: Observer<String, NoError>
+
+    // MARK: Lifecycle
+
+    init() {
+        let (alertMessageSignal, alertMessageObserver) = Signal<String, NoError>.pipe()
+        self.alertMessageSignal = alertMessageSignal
+        self.alertMessageObserver = alertMessageObserver
+    }
+
+    func alertMessageForErrors(errors: [NSError]) -> String {
+        return errors.map({
+            var alertMessage = ""
+            if let failureReason = $0.userInfo[NSLocalizedFailureReasonErrorKey] as? String {
+                alertMessage += failureReason
+            }
+            if let localizedDescription = $0.userInfo[NSLocalizedDescriptionKey] as? String {
+                alertMessage += ": \(localizedDescription)"
+            }
```

Comment:
Maybe we should also change the order of `localizedFailureReason` and `localizedDescription`, since the first one is optional, but the second one isn't. When the first one is `nil`, we have error messages starting with a colon.